### PR TITLE
update news route with others pictures and update news page & components

### PR DESCRIPTION
**Pull request by Gonaline**

### CODE REVIEW

**Review by FrancoisSalmon — APPROVED** [review]
ok for me

File: frontend/src/pages/NewsDetail/index.jsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ function NewsDetail() {
   function date(dm) {
     return `${DateTime.fromISO(dm)
       .setLocale("fr")
-      .toFormat("dd MMM")
+      .toFormat("dd MMMM yyyy")
```
Comment (LuigiMenez) [inline frontend/src/pages/NewsDetail/index.jsx:23]: A month in four figures?

**Review by LuigiMenez — APPROVED** [review]
Ok for me.

File: frontend/src/components/CardNews/index.jsx [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 import PropTypes from "prop-types";
 import SCardNews from "./style";
 
-export default function CardNews({ title, content, img, id }) {
+export default function CardNews({ title, content, img1, id }) {
+  const navigate = useNavigate();
+
+  function changeLocation(placeToGo) {
+    navigate(placeToGo, { replace: true });
+    window.location.reload();
```
Comment (LoicBrassart) [inline frontend/src/components/CardNews/index.jsx:11]: Wait, what ?! o.O

**Review by LoicBrassart — APPROVED** [review]
SnE

**Pull request merged**